go.mod

```diff
@@ -542,3 +542,7 @@ require (
 )
 
 replace github.com/ma314smith/signedxml v1.1.1 => github.com/moov-io/signedxml v1.1.1
+
+// Support using the forked repository until https://github.com/pires/go-proxyproto/pull/110 merges
+// and is released.
+replace github.com/pires/go-proxyproto v1.0.0 => github.com/peteski22/go-proxyproto v1.0.0
```

Comment:
Info on [forked repo tag](https://github.com/peteski22/go-proxyproto/releases/tag/v1.0.0).

> Keep listener listening if upstream connection address is not trusted (but close that connection).
> Releases the changes requested on pires's go-proxyproto by https://github.com/pires/go-proxyproto/pull/110.

Bumped to v1.0.0 which doesn't exist as a tag on the maintainer's repo but does on the forked repo.

Reply:
The change here can now be updated and the `replace` removed, as the PR was accepted, merged and tagged:

https://github.com/pires/go-proxyproto/releases/tag/v0.8.0

![image](https://github.com/user-attachments/assets/eb45feb9-915f-458f-8681-15ad8c2b84f6)